language/dotnet/build-images.md

```diff
@@ -0,0 +1,307 @@
+
+## Overview
+
+Now that we have a good overview of containers and the Docker platform, let’s take a look at building our first image. An image includes everything needed to run an application - the code or binary, runtime, dependencies, and any other file system objects required.
+
+To complete this tutorial, you need the following:
+
+- .NET SDK version 6.0 or later. [Download .NET SDK](https://dotnet.microsoft.com/download){: target="_blank" rel="noopener" class="_"}.
+- Docker running locally. Follow the instructions to [download and install Docker](../../get-docker.md).
+- An IDE or a text editor to edit files. We recommend using [Visual Studio Code](https://code.visualstudio.com/){: target="_blank" rel="noopener" class="_"}.
+
+## Sample application
+
+For our sample application, let’s create a simple application from a template using .NET. Create a directory in your local machine named `dotnet-docker`. Open a terminal and change to that directory. Run the following `dotnet new` command to create a C# app using the ASP.NET Core Web App template.
+
+```console
+$ mkdir dotnet-docker
+$ cd dotnet-docker
+$ dotnet new webApp -n myWebApp -o src --no-https
+```
+
+Output similar to the following appears.
+
+```console
+The template ASP.NET Core Web App was created successfully.
+This template contains technologies from parties other than Microsoft, see https://aka.ms/aspnetcore/6.0-third-party-notices for details.
+```
+
+The command will create a new directory called `src`. View the `src` directory and verify the contents. You should see the following directories and files.
+
+```shell
+├── Pages
+│ ├── Error.cshtml
+│ ├── Error.cshtml.cs
+│ ├── Index.cshtml
+│ ├── Index.cshtml.cs
+│ ├── Privacy.cshtml
+│ ├── Privacy.cshtml.cs
+│ ├── Shared
+│ ├── _ViewImports.cshtml
+│ └── _ViewStart.cshtml
+├── Program.cs
+├── Properties
+│ └── launchSettings.json
+├── appsettings.Development.json
+├── appsettings.json
+├── myWebApp.csproj
+├── obj
+│ ├── myWebApp.csproj.nuget.dgspec.json
+│ ├── myWebApp.csproj.nuget.g.props
+│ ├── myWebApp.csproj.nuget.g.targets
+│ ├── project.assets.json
+│ └── project.nuget.cache
+└── wwwroot
+├── css
+├── favicon.ico
+├── js
+└── lib
+```
+
+## Test the application
+
+Let’s start our application and make sure it’s running properly. Open your terminal and navigate to the `src` directory and use the `dotnet run` command.
+
+```console
+$ cd /path/to/dotnet-docker/src
+$ dotnet run –urls http://localhost:5000
+```
+
+Output similar to the following appears.
+
+```console
+Building...
+warn: Microsoft.AspNetCore.DataProtection.Repositories.FileSystemXmlRepository[60]
+    Storing keys in a directory '/root/.aspnet/DataProtection-Keys' that may not be persisted outside of the container. Protected data will be unavailable when the container is destroyed.
+warn: Microsoft.AspNetCore.Server.Kestrel[0]
+     Unable to bind to http://localhost:5000 on the IPv6 loopback interface: 'Cannot assign requested address'.
```

Comment:
This error seems odd.

For me the app always listened only on port 5016
```
PS C:\Users\stefan\dotnet-docker\src> dotnet run –urls http://localhost:5000
Building...
info: Microsoft.Hosting.Lifetime[14]
      Now listening on: http://localhost:5016
info: Microsoft.Hosting.Lifetime[0]
      Application started. Press Ctrl+C to shut down.
info: Microsoft.Hosting.Lifetime[0]
      Hosting environment: Development
info: Microsoft.Hosting.Lifetime[0]
      Content root path: C:\Users\stefan\dotnet-docker\src\
info: Microsoft.Hosting.Lifetime[0]
```
so in the end I just skipped the `-urls` parameter and still have the app running on port 5016. 🤔

Reply:
Thanks, Stefan. I missed a dash in the flag. It should be `--urls`.  I'm surprised it doesn't give an error. 
As for the output, it was copied from the original blog post. When I run the command, I get the same output as you. I'm on Windows too, but it looks like the blog writer wasn't. Maybe the error has to do with the operating system.
I'll update the flag and the output.